checkov/terraform/checks/resource/panos/PolicyDescription.py

```diff
@@ -0,0 +1,46 @@
+from checkov.terraform.checks.resource.base_resource_check import BaseResourceCheck
+from checkov.common.models.enums import CheckResult, CheckCategories
+
+
+class PolicyDescription(BaseResourceCheck):
+    def __init__(self):
+        name = "Ensure description is populated within security policies"
+        id = "CKV_PAN_8"
+        supported_resources = ['panos_security_policy','panos_security_rule_group']
+        categories = [CheckCategories.NETWORKING]
+        super().__init__(name=name, id=id, categories=categories, supported_resources=supported_resources)
+
+    def scan_resource_conf(self, conf):
+    
+        # Check there is a rule defined in the resource
+        if 'rule' in conf:
+
+            # Report the area of evaluation
+            self.evaluated_keys = ['rule']
+
+            # Get all the rules defined in the resource
+            rules = conf.get('rule')
+
+            # Iterate over each rule
+            for secrule in rules:
+
+                # Check if a description is defined in the rule
+                if 'description' in secrule:
+
+                    # If a description is defined, get the value
+                    desc = secrule.get('description')
+
+                    if desc[0] == "":
```

Comment:
```suggestion
                    if desc[0].strip() == "":
```
how about adding `strip` here, then we can even trick the sneaky ones, who just add a space or multiple 😄

Reply:
Yes, I like your suggestion 👍